data/styles/default/taginfo.json

```diff
@@ -0,0 +1,5038 @@
+      "object_types": ["node"],
+      "value": "maze",
+      "key": "attraction"
+    },
+    {
+      "icon_url": "https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/default/light/symbols/playground-m.svg",
+      "object_types": ["node"],
+      "value": "bumper_car",
+      "key": "attraction"
+    },
+    {
+      "icon_url": "https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/default/light/symbols/animal-m.svg",
+      "object_types": ["node"],
+      "value": "animal",
+      "key": "attraction"
+    },
+    {
+      "icon_url": "https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/default/light/symbols/playground-m.svg",
+      "object_types": ["node"],
+      "value": "carousel",
+      "key": "attraction"
+    },
+    {
+      "icon_url": "https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/default/light/symbols/playground-m.svg",
+      "object_types": ["node"],
+      "value": "amusement_ride",
+      "key": "attraction"
+    },
+    {
+      "icon_url": "https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/default/light/symbols/car-repair-m.svg",
+      "object_types": ["node"],
+      "value": "tyres",
+      "key": "service"
+    },
+    {
+      "icon_url": "https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/default/light/symbols/car-repair-m.svg",
+      "object_types": ["area"],
+      "value": "tyres",
+      "key": "service"
+    },
+    {
+      "icon_url": "https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/default/light/symbols/car-repair-s.svg",
+      "object_types": ["area"],
+      "value": "tyres",
+      "key": "service"
+    },
+    {
+      "icon_url": "https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/default/light/symbols/car-repair-s.svg",
+      "object_types": ["node"],
+      "value": "tyres",
+      "key": "service"
+    },
+    {
+      "icon_url": "https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/default/light/symbols/parking-disabled-m.svg",
+      "object_types": ["node"],
+      "value": "disabled",
+      "key": "parking_space"
+    },
+    {
+      "icon_url": "https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/default/light/symbols/parking-disabled-m.svg",
+      "object_types": ["area"],
+      "value": "disabled",
+      "key": "parking_space"
+    },
+    {
+      "icon_url": "https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/default/light/symbols/drinking-water-no-s.svg",
+      "object_types": ["node"],
+      "value": "not",
+      "key": "drinking_water"
+    },
+    {
+      "icon_url": "https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/default/light/symbols/drinking-water-no-m.svg",
+      "object_types": ["node"],
+      "value": "not",
+      "key": "drinking_water"
+    },
```

Comment:
might need to hardcode removing this? IIRC it's converted to `not` because the generator can't handle `no` properly yet.

Reply:
Thanks for reviewing the PR, @RedAuburn! I have improved the code a lot and now it contains 6000+ lines and without duplicates.

What is this generator and why cannot it output "no" values instead of "not"?